tasks/98_linked-list/linked-list.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Represents linked list data structure
+ */
+export class LinkedList {
+  /**
+   * creates new empty linked list
+   */
+  constructor() {
+    this.head = null;
+    this.tail = null;
```

Comment:
Please, read about linked lists. It doesn't contain a link to the tail. Only head.

Reply:
It wasn't my idea, tests do actually require tail to be an actual property of the linked list